pages/linux/ip-rule.md

```diff
@@ -0,0 +1,36 @@
+# ip rule
+
+> IP Routing policy database management.
+> More information: <https://man7.org/linux/man-pages/man8/ip-rule.8.html>.
+
+- Display the routing policy:
+
+`ip rule {{show|list}}`
```

Comment:
If those are exactly the same I think we should just specify one to avoid confusion

Reply:
Oh. I based a similar command at https://github.com/tldr-pages/tldr/blob/main/pages/linux/ip-route.md
Do you think I should modify it to show or list ?

Reply:
No problem, this isn't any fixed rule. Let's just wait for some more opinions

Reply:
Hey @navarroaxel , What do you think about this ?

Reply:
Hmmm. Normally I'd say pick 1 and stick with, but in this case since the other page is doing it I'd go for consistency.

Reply:
Okay, I'll keep it as it is for consistency